packages/acceptit-Tests.package/ACCreateUserStoryTest.class/instance/testItHasCorrectClassName.st

```diff
@@ -1,3 +1,3 @@
 tests
 testItHasCorrectClassName
```

Comment:
This propably needs to be changed to testItHasCorrectClassTitle? But actually its rather a class name, so how should we deal with it?

Reply:
The title and the classname are two different things. One is the class name of the UserStory class, the other is the title of the actual userStory. The classname is generated from the title.